src/pages/home/report/ReportActionItem.js

```diff
@@ -632,19 +632,19 @@ function ReportActionItem(props) {
                             </ReportActionItemSingle>
                             <View style={styles.reportHorizontalRule} />
                         </View>
-                    </>
+                    </View>
                 );
             }
             return (
-                <>
+                <View style={[StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth, true)]}>
```

Comment:
The meaning of 2nd param is `isMoneyReport`. Why is set to true here? (this is task report)

Reply:
I wanted to reuse the function in order to increase the `topMargin`. The task report requires it in order to move the background screen a bit higher. I can rename it to be eg. `MONEY_OR_TASK_REPORT`, if you wish

Reply:
ok let's rename then

Reply:
done :)